Documentation/ContentMapping/Index.rst

```diff
@@ -1,186 +1,372 @@
-=================================
+Content rendering via data processing
+=====================================
 
-To use the TypoScript, it needs to be loaded (*included*), so open file
-`Configuration/TypoScript/setup.typoscript` and add line
-`@import '...'` as shown here in the second line::
+Just like with the menu, the content can also be displayed by using
+a data processor, the :php:`DatabaseQueryProcessor`.
 
-   @import 'EXT:fluid_styled_content/Configuration/TypoScript/setup.typoscript'
-   @import 'EXT:site_package/Configuration/TypoScript/Helper/DynamicContent.typoscript'
+Define the data processor in :typoscript:`page.10.dataProcessing` beside the
+data processors of the menu::
 
-   page = PAGE
    page {
-      // ...
+     10 {
+       dataProcessing {
+         //10 = TYPO3\CMS\Frontend\DataProcessing\MenuProcessor
+         20 = TYPO3\CMS\Frontend\DataProcessing\DatabaseQueryProcessor
+         20 {
+           table = tt_content
+           orderBy = sorting
+           where = colPos = 0
+           as = mainContent
+         }
+       }
+     }
    }
 
-   config {
-      // ...
-   }
+In the Fluid template there will be a variable called "mainContent" available,
+containing an array of all contents of this column on the current page.
 
+We need one data processor for each column so lets add the other two::
+
+   page {
+     10 {
+       dataProcessing {
+         //10 = TYPO3\CMS\Frontend\DataProcessing\MenuProcessor
+         //20 = TYPO3\CMS\Frontend\DataProcessing\DatabaseQueryProcessor
+         30 = TYPO3\CMS\Frontend\DataProcessing\DatabaseQueryProcessor
+         30 {
+           table = tt_content
+           orderBy = sorting
+           where = colPos = 1
+           as = jumbotronContent
+         }
+         40 = TYPO3\CMS\Frontend\DataProcessing\DatabaseQueryProcessor
+         40 {
+           table = tt_content
+           orderBy = sorting
+           where = colPos = 2
+           as = sidebarContent
+         }
+       }
+     }
+   }
 
 .. _cm-fluid-typoscript-mapping:
 
-Typoscript Mapping in Fluid Template
+TypoScript mapping in Fluid template
 ====================================
 
 .. highlight:: html
 
-Open file :file:`Resources/Private/Templates/Page/Default.html` and locate the
-three columns. They all show a "Headline" (look for the :code:`<h2>`-tags) and some
-dummy content (look for the :code:`<p>`-tags).
+Open the file :file:`Resources/Private/Templates/Page/Default.html` and locate the
+main content area. It contains a headline (look for the :code:`<h2>`-tags) and
+some dummy content (look for the :code:`<p>`-tags).
 
 Simply replace these lines with the cObject-ViewHelper (`<f:cObject ... >`),
-so that file :file:`Default.html` shows the following HTML code. Make sure, you
-specify the column positions correctly (:code:`1`, :code:`0` and :code:`2`) and
-in exactly this order::
+so that file :file:`Default.html` shows the following HTML code.
+
+.. code-block:: html
 
```

Comment:
Maybe you want to use the short `<html>` opening tag from https://github.com/TYPO3/typo3/blob/master/typo3/sysext/fluid_styled_content/Resources/Private/Templates/Textmedia.html#L1?
That gives a sane place for custom ViewHelper namespaces later.

Reply:
I will replace all code examples by extracts form the extension anyway.